value/processor/pom.xml

```diff
@@ -40,15 +40,15 @@
   </scm>
 
   <properties>
-    <auto-service.version>1.1.1</auto-service.version>
+    <auto-service.version>999.0.0</auto-service.version>
```

Comment:
It might just be that we need to change this one.

We do need to change _something_ so that the version starts with a number: https://github.com/google/truth/pull/1601#discussion_r2366370155

Reply:
Is this property meant to point to the latest released version? If so, I will fix this now.

Reply:
That is my recollection, at least.

(Since the latest released version isn't modularized, that may complicate matters.)